### ISSUE TITLE
Run svgcheck with -N

### ISSUE THREAD

**Issue by larseggert** [cabo/kramdown-rfc2629#107]
`svgcheck` seems to sometimes die when the network is not available, see https://github.com/larseggert/i-d-toolchain/runs/1969937318?check_suite_focus=true#step:4:51

I wonder if it would make sense to run it with `-N` aka `--no-network`?

(I was surprised it needed to make network connections to begin with.)

**Comment by larseggert** [comment]
So it turns out that `-N` prevents any network requests, but that basically makes `svgcheck` fail for me, because it then says it can't retrieve `http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd` anymore.

Could you remove the `-N` again?

**Comment by cabo** [comment]
Weird, I just used svgcheck with 1.4.2 and it worked for me.  What was the input?

**Comment by cabo** [comment]
(I'm using svgcheck 0.6.0)

**Comment by larseggert** [comment]
I only see this when I run inside GitHub Actions. I wonder if they have a weird network setup.

**Comment by larseggert** [comment]
For example, this fails, and has been failing since you made the change 13 days ago: https://github.com/larseggert/i-d-toolchain/runs/2380758454?check_suite_focus=true#step:6:36

**Comment by cabo** [comment]
I notice that I have svg11.dtd in my ~/.cache/xml2rfc
Renaming it doesn't cause the problem for my current I-D, though.
It seems that your test fails at *** could not create svg for "\nmsc {\n a [label=...
It also seems that the download to cache of svg11.dtd and all its dependencies was initiated when I processed an mscgen file a couple of months ago.

**Comment by larseggert** [comment]
Yeah, I don't really know what `svgcheck` is doing. All I noticed is that the issue doesn't happen when I remove the `-N` again.

**Comment by cabo** [comment]
Yep, mscgen spits out

<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN"
 "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">

and svgcheck then wants to read that (I moved my cached copy away).

**Comment by cabo** [comment]
I pushed a 1.4.3 without -N
A better fix would be to remove the DOCTYPE from the mscgen output, but that seems a larger effort.
